### PR TITLE
on_missing ignore to skip and foo

### LINKED ISSUE #77: on_missing: "ignore" -> "skip"
Rename `on_missing="ignore"` to `on_missing="skip"` to be consistent with #60. Do a deprecation advising to use the new one. Also raise an error if an invalid one is passed.

As an example, see https://github.com/regionmask/regionmask/pull/246/files, where the `proj` keyword was renamed to `projection` (among others).

### PR DESCRIPTION
**Pull request by veni-vidi-vici-dormivi**
Closes #77 
Change option "ignore" to "skip" in on_missing to be consistent with on_parse_error (see #75).
Implement behavior if on_missing is none of the implemented options.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 ## v0.3.0 - unreleased
 
+- Change on_missing option "ignore" to "skip". ([#79](https://github.com/mathause/filefinder/pull/79))
```
Comment (mathause) [inline CHANGELOG.md:5]:
```suggestion
- Change `on_missing` option in the `priority_filter` from "ignore" to "skip".
  ([#79](https://github.com/mathause/filefinder/pull/79))
```

**Review by mathause — COMMENTED** [review]
Looks good. The check for the correct keyword is often done outside the inner `if` to fail early (the disadvantage is that the valid keywords are now defined in two places). 

See e.g.: https://github.com/regionmask/regionmask/blob/8a1923d3dbfd83d2872e6067d128f7e349520ea9/regionmask/core/plot.py#L338

**Comment by mathause** [comment]
> Would you still put it in the function where it is actually used or in the wrapper?

Here, I would put it in `priority_filter` (however, I don't want to generalize. If `_prioritize` was used in several places its prob. better to put it there)

**Comment by veni-vidi-vici-dormivi** [comment]
Actually in on_parse_error the options are **'raise'**, 'warn' and 'skip'. I guess we should change 'error' to 'raise' here too? 😅

**Comment by mathause** [comment]
> Actually in on_parse_error the options are **'raise'**, 'warn' and 'skip'. I guess we should change 'error' to 'raise' here too? 😅

Good catch. I quickly had a look (in xarray) and `"raise"` seems the default choice. 

(I also found that they rather use `"ignore"` instead of `"skip"`, so I am considering if we really should not rather use `"skip"` here)

**Comment by veni-vidi-vici-dormivi** [comment]
But for parse_error skip -> ignore we don't need a FutureWarning because we didn't release it yet, is that right?

**Review by mathause — APPROVED** [review]
LGTM

**Comment by mathause** [comment]
Can you merge main into this PR? Maybe you'll get a merge conflict :rocket:

**Pull request merged**